tests/integration/test_basket_waitlist_subscription.py

```diff
@@ -18,9 +18,9 @@ class Settings(BaseSettings):
     # We initialize CTMS api client id/secret in `ctms-db-init.sql`
     ctms_client_id: str = "id_integration-test"
     ctms_client_secret: str
-
-    class Config:
-        env_file = os.path.join(TEST_FOLDER, "basket.env")
+    model_config = SettingsConfigDict(
+        env_file=os.path.join(TEST_FOLDER, "basket.env"), extra="ignore"
```

Comment:
Without `extra="ignore"`, any environment variable not specified in these settings triggers a validation error